.github/workflows/release-charts.yaml

```diff
@@ -34,7 +35,7 @@ jobs:
           echo "${{ secrets.GITHUB_TOKEN }}" | helm registry login ghcr.io -u $ --password-stdin
           echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
       
-      - name: Build, Push and Sign chart
+      - name: Build, Push and Sign legacy chart
```

Comment:
https://github.com/microsoft/retina/issues/1115

Reply:
I was planning to leave this as legacy since the directory is still named legacy - and then update this reference, once there is consensus on the naming. thoughts?